specs/protocol/exec-engine.md

```diff
@@ -514,7 +514,7 @@ For the Ecotone upgrade, this entails that:
 [engine_getPayloadV2]: https://github.com/ethereum/execution-apis/blob/584905270d8ad665718058060267061ecfd79ca5/src/engine/shanghai.md#engine_getpayloadv2
 [engine_getPayloadV3]: https://github.com/ethereum/execution-apis/blob/a0d03086564ab1838b462befbc083f873dcf0c0f/src/engine/cancun.md#engine_getpayloadv3
 [engine_getPayloadV4]: https://github.com/ethereum/execution-apis/blob/869b7f062830ba51a7fd8a51dfa4678c6d36b6ec/src/engine/prague.md#engine_getpayloadv4
-[HEX value encoding]: https://eth.wiki/json-rpc/API#hex-value-encoding
+[HEX value encoding]: https://ethereum.org#hex-value-encoding
```

Comment:
It looks like neither of this links work anymore

Reply:
For me `https://ethereum.org#hex-value-encoding` is not working, it seems to be missing the path. That's the landing page and a non-existent anchor.